consolidate docs

ajm188 opened the pull request:
docs for https://github.com/vitessio/vitess/pull/13959

Commit:
regen consolidated docs

Signed-off-by: Andrew Mason <andrew@planetscale.com>

Commit:
remove handwritten vtctld/vtorc docs

Signed-off-by: Andrew Mason <andrew@planetscale.com>

deepthi reviewed (approved):
nit: with this structure Hugo produces a `>` after each program in the left nav whether are not there are any sub-sections. There are additional docs for `vtctl` and `vtctldclient` but not for `vtctld` and `vtorc`. Could be misleading when people are just browsing docs.

Commit:
no need to manage checkouts if ref is HEAD

Signed-off-by: Andrew Mason <andrew@planetscale.com>

Commit:
add ability to generate all docs with one target

Signed-off-by: Andrew Mason <andrew@planetscale.com>

Commit:
fix sidebar expanders

this makes it so we only expand if there are actual "regular pages" (i.e. not just `_index.md`) in a section

Signed-off-by: Andrew Mason <andrew@planetscale.com>

ajm188 commented:
okay, i think [707456b](https://github.com/vitessio/website/pull/1583/commits/707456bec40a5abf40a4c021dbd87970cccc22a0) fixes it. i tested in on the https://github.com/vitessio/website/pull/1582 and it had expanders only for `mysqlctl`, `vtctl`, and `vtctldclient` and none of the others
